<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - Test</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>Test</h1>
<p>
  Test duration: 0.044s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">regression.TestAssert</td>
      </tr>
            
<tr>
  <td class="method">
                  testAssert
      </td>
  <td class="duration">
    0.004s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="test.png" target="_blank">screenshot link</a>
                  </div>
    

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected:&lt;pass&gt; but was:&lt;fail&gt;</b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)<br />
              org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)<br />
              regression.TestAssert.testAssert(TestAssert.java:17)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:498)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  testAssert1
      </td>
  <td class="duration">
    0.005s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="test.png" target="_blank">screenshot link</a>
                  </div>
    

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: test1 expected:&lt;test1&gt; but was:&lt;this has passed&gt;</b></a><br />
    <div class="stackTrace" id="exception-1">
              org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)<br />
              regression.TestAssert.testAssert1(TestAssert.java:26)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:498)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">testcases.TestAssert</td>
      </tr>
            
<tr>
  <td class="method">
                  testAssert1
      </td>
  <td class="duration">
    0.010s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="test.png" target="_blank">screenshot link</a>
                  </div>
    

        <a href="javascript:toggleElement('exception-2', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [fail] but found [pass]</b></a><br />
    <div class="stackTrace" id="exception-2">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertEqualsImpl(Assert.java:137)<br />
              org.testng.Assert.assertEquals(Assert.java:118)<br />
              org.testng.Assert.assertEquals(Assert.java:453)<br />
              org.testng.Assert.assertEquals(Assert.java:463)<br />
              testcases.TestAssert.testAssert1(TestAssert.java:15)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:498)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">testcases.TestAssertions</td>
      </tr>
            
<tr>
  <td class="method">
                  testAssert4
      </td>
  <td class="duration">
    0.002s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="test.png" target="_blank">screenshot link</a>
                  </div>
    

        <a href="javascript:toggleElement('exception-3', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: this is failing expected [false] but found [true]</b></a><br />
    <div class="stackTrace" id="exception-3">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertEqualsImpl(Assert.java:137)<br />
              org.testng.Assert.assertEquals(Assert.java:118)<br />
              org.testng.Assert.assertEquals(Assert.java:568)<br />
              testcases.TestAssertions.testAssert4(TestAssertions.java:25)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:498)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
      </table>


  <table class="resultsTable">
    <tr><th colspan="3" class="header passed">Passed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">testcases.TestAssert</td>
      </tr>
            
<tr>
  <td class="method">
                  testAssert2
      </td>
  <td class="duration">
    0.003s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">testcases.TestAssertions</td>
      </tr>
            
<tr>
  <td class="method">
                  testAssert3
      </td>
  <td class="duration">
    0.002s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
      </table>
  
</body>
</html>
